README.md

```diff
@@ -29,6 +83,9 @@ expect(tree).toMatchStyledComponentsSnapshot()
 
 ## toHaveStyleRule [React]
 
+Only checks for the styles directly applied to the component it receives, to assert that a style has been applied to a child
```

Comment:
> to assert that a style has been applied to a child component

You can make assertion on a child component using Enzyme's [find](https://github.com/airbnb/enzyme/blob/master/docs/api/ReactWrapper/find.md).
Snapshots should be used for testing [complex selectors](https://github.com/styled-components/jest-styled-components/issues/33#issue-237062310) and/or the entire output in one go.